VisualPinball.Unity/VisualPinball.Unity/VPT/Table/TableSerializedContainer.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using VisualPinball.Engine.VPT;
+using VisualPinball.Engine.VPT.Table;
+
+namespace VisualPinball.Unity.VPT.Table

```

Comment:
Change to `VisualPinball.Unity`

Reply:
Same, placed next to existing files, do i have to clean that up ?

Reply:
Leaving them will make it easier when I merge the other branch :)